File: c/2-control structures/practice/csw-5.c
#include <stdio.h>
int main()
{
    float a;
    scanf("%f\n",&a);
    if(a>=1&&a<=5)
        printf("100 USD\n");
    else if(a>5&&a<=20)
        printf("200 USD\n");
    else if(a>20&&a<=50)
        printf("300 USD\n");
    else
        printf("Invalid Input");
}

File: python/3-conditional statements/workout/csw-5.py
try:
    num1 = float(input())
    num2 = float(input())
    num3 = float(input())

    if num1 < 0 or num2 < 0 or num3 < 0:
        print("Invalid input")
    else:
        if num1 >= num2 and num1 >= num3:
            print(num1)
        elif num2 >= num1 and num2 >= num3:
            print(num2)
        else:
            print(num3)

except ValueError:
    print("Invalid input")